synapseclient/table.py

```diff
@@ -567,7 +568,11 @@ class EntityViewSchema(SchemaBase):
     :param parent: the project in Synapse to which this table belongs
     :param scopes: a list of Projects/Folders or their ids
     :param view_type: the type of EntityView to display: either 'file' or 'project'. Defaults to 'file'
-    :param add_default_columns: whether to add the default view columns based on the EntityView. Defaults to True. 
+    :param addDefaultViewColumns: whether to add the default view columns based on the EntityView. Defaults to True.
+    :param addAnnotationColumns: whether to create columns based on set union of all annotations defined in the EntityViewSchema's scope. Defaults to True.
```

Comment:
whether to add all default columns (e.g. name, createdOn, modifiedBy etc) defaults to True

Reply:
whether to add columns for all annotation keys defined across any entity in the scope.